helper_posix_test.go

```diff
@@ -23,16 +23,7 @@ func helloGoodbye() [][]string {
 }
 
 func rawHelloGoodbye() string {
-	return `"command": [
-    [
-      "echo",
-      "hello world!"
-    ],
-    [
-      "echo",
-      "goodbye world!"
-    ]
-  ]`
+	return `["echo", "hello world!"], ["echo", "goodbye world!"]`
```

Comment:
Note, there are 9 places that call `rawHelloGoodbye()` in `payload_test.go` that will need to be updated to expect the new format.